scene/gui/dialogs.cpp

```diff
@@ -272,8 +282,11 @@ int WindowDialog::_drag_hit_test(const Point2 &pos) const {
 
 void WindowDialog::set_title(const String &p_title) {
 
-	title = tr(p_title);
-	update();
+	String new_title = tr(title);
```

Comment:
Shouldn't this be `String new_title = tr(p_title);`?

Reply:
Oops, copy-paste error...